guide/README.md

```diff
@@ -382,6 +382,25 @@ block `re` is lowered by one half step; on the right, both *Pitch*
 blocks are raised by one half step. (You can also use a double-sharp
 or double-flat accidental.) [RUN LIVE](https://musicblocks.sugarlabs.org/index.html?id=1733231694757697&run=True)
 
+![double sharp and double flat](./double-sharp-flat.svg "Using Double Sharp and Double Flat blocks")
+
+
+Double sharps and flats have a special place in music theory. They are used to maintain diatonic structure in music, which means for the key, each letter is used once.
+
+For example, in the C major scale, it goes C - D - E - F - G - A - B. Now for C# major, it's the same thing, with C# - D# - E# - F# - G# - A# - B#. If you want to simplify the amount of sharps in this, you can write C# – D# – F – F#- G# – A# – C, which has two F's and 2 C's, which makes it more confusing to read. To prevent this, we use a diatonic structure.
```

Comment:
Perhaps, instead of "If you want to simplify the amount of sharps in this," you say: "Another way to write this could have been"